### DISCUSSION TITLE
Welcome to akamai_v2_toolkit Discussions!

### DISCUSSION THREAD

**Discussion by luluhoc** [luluhoc/akamai_v2_toolkit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cbxhpy** [comment]
how to use this projiect ?

**Comment by wolf427** [comment]
Hi, I can't find your account on Twitter. Could you please tell me how to bypass akaimai detection? I'm also a programmer working on this problem. Thank you very much.
